site/_guides/ingressroute-to-httpproxy.md

```diff
@@ -140,7 +140,7 @@ spec:
 
 HTTPProxy offers additional ways to match incoming requests to routes.
 This document covers the conversion between the routing offered in IngressRoute and HTTPProxy.
-For a broader discussion of HTTPProxy routing, see the [Routing section of the HTTPProxy documentation]({% link _docs_1_0/httpproxy.md %}#routing).
+For a broader discussion of HTTPProxy routing, see the [Routing section of the HTTPProxy documentation](/docs/1.0.0/httpproxy.md#routing).
```

Comment:
Is it possible to use the link syntax? I like that it gives us some notification if we move a file and it breaks a link.

Reply:
I've updated all of them which didn't move to /docs. I need to come up with a way to make the syntax dynamic which I think might be able to go into a new PR if that's acceptable.